### ISSUE TITLE
Storage Widget not update correctly

### ISSUE THREAD

**Issue by ivandiazmolina** [IceWhaleTech/CasaOS#98]
**Describe the bug**
I have a Raspberry PI 4 with a 16 GB SD card and a 1TB hard disk connected by usb. I have the CASAOS operating system installed on the SD and I have also installed nextcloud on the hard drive to store my files. The problem is that the hard drive widget always shows "Used: N/A" but it is taking up space. It's a minor bug but it's to contribute to improve CASAOS.

Best regards

**To Reproduce**
Simply put files into external HDD and check in the Home Screen the widget

**Expected behavior**
The plugin should update the information about the hard disk storage.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: macOS Monterey
 - Browser: Safari
 - Version CasaOS 0.2.8

**Additional context**

**Comment by LinkLeong** [comment]
The USB device is automatically mounted. If the mounting fails due to the partition format or non partition of the disk, N/A will be displayed

**Comment by ivandiazmolina** [comment]
The partition format is Ext4, isn’t correct? Because the hard disk is working nice

**Comment by LinkLeong** [comment]
Hi,Can you run `lsblk -o name,size,type,fstype` and take a screenshot and send it to me?

**Comment by ivandiazmolina** [comment]
<img width="446" alt="Captura de pantalla 2022-02-07 a las 9 18 33" src="https://user-images.githubusercontent.com/5048460/152750357-ffe84e19-66ad-485c-8703-6eed4e9f1600.png">

**Comment by LinkLeong** [comment]
No, your sda is not partitioned, so it will not show data.
![image](https://user-images.githubusercontent.com/13556972/152752953-d6ac0c38-0d74-4670-8117-fd2497f75d19.png)

**Comment by ivandiazmolina** [comment]
I'm going to see how to partition the disk without losing the information. Thank you!
